CHANGELOG.md

```diff
@@ -7,6 +7,16 @@
 
 ### Features
 - Add grants to materializations ([dbt-labs/dbt-spark#366](https://github.com/dbt-labs/dbt-spark/issues/366), [dbt-labs/dbt-spark#381](https://github.com/dbt-labs/dbt-spark/pull/381))
+- Add `connection_parameters` for databricks-sql-connector connection parameters ([#135](https://github.com/databricks/dbt-databricks/pull/135))
+    - This can be used to customize the connection by setting the additional parameters.
+    - The full parameters are listed at [Databricks SQL Connector for Python](https://docs.databricks.com/dev-tools/python-sql-connector.html#connect-method).
+    - Currently, the following parameters are reserved for `dbt-databricks`. Please use the normal credential settings instead.
```

Comment:
From the documentation, it seems that the only parameter users can set is `http_headers`? I do see a list of internal arguments that can be set as well https://github.com/databricks/databricks-sql-python/blob/161b94e9c7832fa8393cdff7bc9eac14f6ba840e/src/databricks/sql/client.py#L50

Reply:
Yeah, actually the purpose here includes them, but I'm not sure we should mention the internal parameters here.

Reply:
Yea we don't need to mention the internal ones. Just realized the only one that can be set is http_headers (maybe there will be more in the future).